adam/permission.py

```diff
@@ -11,14 +11,14 @@ def __init__(self, right, target_type, target_uuid):
         
         Args:
             right: must be one of "READ", "WRITE", "ADMIN", "GRANT_READ", "GRANT_WRITE"
-            target_type: must be one of "PROJECT", "BATCH", "SETTINGS", "GROUP"
```

Comment:
Deleting "SETTINGS" makes the unit test fail (`test_get_method` in `permission_test.py`)

Reply:
dang it, oops. fixed.